CMakeLists.txt

```diff
@@ -22,7 +22,30 @@ set_env( MOBILEID_URL "https://dd-mid.ria.ee/mid-api" CACHE STRING "URL for Mobi
 set_env( SMARTID_URL "https://dd-sid.ria.ee/v1" CACHE STRING "URL for Smart-ID" )
 set(CMAKE_INTERPROCEDURAL_OPTIMIZATION YES)
 set(CMAKE_INTERPROCEDURAL_OPTIMIZATION_DEBUG NO)
-
+set(CPACK_PACKAGE_VERSION ${VERSION})
+set(CPACK_GENERATOR "RPM")
+set(CPACK_PACKAGE_NAME "qdigidoc4")
+set(CPACK_PACKAGE_RELEASE 1)
+set(CPACK_PACKAGE_CONTACT "Andrus Randveer")
+set(CPACK_PACKAGE_VENDOR "RIA")
+set(CPACK_PACKAGING_INSTALL_PREFIX "/usr/")
+set(CPACK_RPM_EXCLUDE_FROM_AUTO_FILELIST_ADDITION
+        "/usr/local" "/usr/local/etc" "/usr/share/man"
```

Comment:
Package should not install anything into /usr/local folder

Reply:
It had some conflicts with the Fedora 32 (maybe 33?) filesystem package, but in 34 it's fine.